todo/src/components/Todos/TodoContent.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+import TodoList from './TodoList.js'
```

Comment:
Component is imported on line 1, can change line 3 to `class TodoContent extends Component {`